test/suites/secondary_redis.bash

```diff
@@ -0,0 +1,95 @@
+SUITE_secondary_redis_PROBE() {
+     if [ "`which redis-server`" = "" ]; then
```

Comment:
On some Ubuntu docker images `which` is not available by default. Better use `[[ -x "$(command -v redis-server)" ]]`.